recipes/java-jdk/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "8.25.17" %}
+
+package:
+  name: java-jdk
+  version: 8.25.17
+
+# Cannot use source section because of cookies blocking download
+#source:
+#  fn: jre-8u25-linux-x64.tar.gz
+#  url: http://download.oracle.com/otn-pub/java/jdk/8u25-b17/jre-8u25-linux-x64.tar.gz
+
+build:
+  number: 0
```

Comment:
Yeah... Unfortunately I don't have access to a Mac and this is no longer on my critical path.  I tried updating the version number.  Hopefully that will help!